srcpkgs/megatools/template

```diff
@@ -1,21 +1,23 @@
 # Template file for 'megatools'
 pkgname=megatools
-version=1.10.3
-revision=3
-build_style=gnu-configure
-hostmakedepends="pkg-config asciidoc"
+version=1.11.0
+revision=1
+_release_date=20220519
```

Comment:
```suggestion
version=1.11.0.20220519
revision=1
```
what would you think about this?

Reply:
I'd prefer to leave the (user facing) version as 1.11.0 because that's how upstream refers to it most of the time, but I don't have any strong feelings about it either way.